opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp

```diff
@@ -125,9 +130,12 @@ namespace Opm
         Tuning m_tuning;
         MessageLimits m_messageLimits;
         Phases m_phases;
+        std::map<int, DynamicState<std::shared_ptr<VFPProdTable>>> vfpprod_tables;
+        std::map<int, DynamicState<std::shared_ptr<VFPInjTable>>> vfpinj_tables;
```

Comment:
Maybe you want to add `m_` to tell they are member variables?  No big deal though.

Reply:
No I don't - I consider the `m_` prefix a mistake; I much prefer using `this->` in the methods.